src/bfx_tools/seq2HLA.ml

```diff
@@ -2,6 +2,14 @@
 open Biokepi_run_environment
 open Common
 
+
+type product = <
+    is_done : Ketrew_pure.Target.Condition.t option ;
+    class1_path : string;
+    class2_path: string;
+    work_dir_path: string >
+
+
 let hla_type ~work_dir ~run_with ~r1 ~r2 ~run_name =
```

Comment:
LGTM but since we don't have an MLI file there; could you force the type with an annotation (`: product KEDSL.workflow_node`)?